src/orientation.rs

```diff
@@ -104,10 +104,10 @@ impl SensorAccelerometer {
 
         // Compute angle to landscape/portrait mode.
         //
-        // This will return the angle between the orientation axis and the position. So
-        // for example when the portrait_angle is zero it means that device is
-        // vertical, however to determine whether it's upside down or vise-versa
-        // we must look at the angle to landscape.
+        // This will return the angle between the orientation axis and the
+        // position. So for example when the portrait_angle is zero it means
+        // that device is vertical, however to determine whether it's upside
+        // down or vice-versa we must look at the angle to landscape.
```

Comment:
```suggestion
        // down or not we must look at the angle to landscape.
```

Just sounds better than vice-versa here.